org.eclipse.scout.rt.server/src/main/java/org/eclipse/scout/rt/server/ServiceTunnelServlet.java

```diff
@@ -128,6 +130,7 @@ protected void doPost(HttpServletRequest servletRequest, HttpServletResponse ser
       m_serverRunContextProducer.get()
           .getInnerRunContextProducer()
           .produce(servletRequest, servletResponse)
+          .withProperty(ServiceTunnelOptions.ID_SIGNATURE_PROP, enableSignature(servletRequest))
```

Comment:
Maybe we could skip setting the property if it is false? Not having the property means the same as false, therefore there is no need to add it to the map in most cases?